src/Selenium2Library/keywords/browsermanagement.py

```diff
@@ -62,6 +65,32 @@ def close_browser(self):
             self.browsers.close()
 
     @keyword
+    def delete_all_cookies_across_all_domains(self):
+        """ Deletes cookies fom all unique domains visited. """
+        if not self.ctx._domain_tracking:
+            raise AssertionError("Cannot use this keyword when domain tracking is not enabled.")
+        self.ctx._domain_tracking = False
+        for domain in self.ctx._domains_visited:
+            self.go_to(domain)
+            self.browser.delete_all_cookies()
+        self.ctx._domains_visited = set()
+        self.ctx_domain_tracking = True
+
+    @keyword
+    def get_domains_visited(self):
+        """ Returns a list of unique domains visited. """
+        if not self.ctx._domain_tracking:
```

Comment:
I think keyword should not fail if feature is not enabled. Retuning just False or None could be better.

Or do you have use case why it should fail?